awx/main/models/events.py

```diff
@@ -536,7 +536,7 @@ def _hostnames(self):
             pass
         return hostnames
 
-    def _update_host_summary_from_stats(self, hostnames):
+    def _update_host_summary_from_stats(self, hostnames, bulk_update_hosts_sorted_by_id):
```

Comment:
It looks like `bulk_update_hosts_sorted_by_id` is your utility method. It would make more sense to import that method as opposed to taking it as an argument.

Guessing, but you might have had problems with circular imports. The thought process should be to keep a mental model or "order" of models. Utility functions should have the fewest (ideally no) internal dependencies, so I would point to my other comment that the module with the utility method shouldn't do `from awx.main.models.inventory import Host` and this should address any circularity you had importing that method from _this_ module.